stock_lot_multi_image/models/stock_lot.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class StockLot(models.Model):
+    _name = "stock.lot"
+    _inherit = ["stock.lot", "base_multi_image.owner"]
+
+    image = fields.Image(compute="_compute_image")
```

Comment:
should we also use `image.mixin` to have the option to store the main image under multiple size, then use `image_1920` instead of `image` as defined here? it maybe useful if someone want to display a thumbnail of the main image

Reply:
@hoangtrann I agree, @GabbasovDinar please add dependency on the image.mixin too.